src/editor/components/scenegraph/Toolbar.js

```diff
@@ -218,48 +183,42 @@ export default class Toolbar extends Component {
         console.log(
           'no urlSceneId or doSaveAs is true, therefore generate new one'
         );
-        currentSceneId = await generateSceneId(this.props.currentUser.uid);
+        currentSceneId = await createScene(
```

Comment:
instead of generating a scene id and then updating a scene, we are simply creating a scene w/ all the relevant info